psi4/src/psi4/libfock/cubature.h

```diff
@@ -56,11 +54,31 @@ class Options;
 //
 // RMP: You got that right. Calling nuclear weights one point at a time is another
 // great way to get a 1000x slowdown. What an incredible smell you've discovered!
-//
+
+
 struct MassPoint {
     double x, y, z, w;
 };
 
+struct SphericalGrid {
+    /// Number of points in radial grid
+    int npoints_;
+    /// Order of spherical harmonics in spherical grid (integrates products up to L_tot = 2 * order_ + 1)
+    int order_;
+};
+
+struct RadialGrid {
+    int npoints_;
+    /// Alpha scale (for user reference)
+    double alpha_;
+    /// Nodes (including alpha)
+    double* r_;
+    /// Weights (including alpha and r^2)
+    double* w_;
```

Comment:
Please do not introduce raw pointers, when their use can easily be avoided. You can just duplicate the data into `std::vector<double>`.

Reply:
Fixed (I think? My C++ is very rusty, if you haven't noticed...).